components/prism-keepalived.js

```diff
@@ -0,0 +1,32 @@
+Prism.languages.keepalived = {
+	'comment': {
+		pattern: /(^|[^"{\\])#.*/,
+		lookbehind: true
+	},
+	'property': {
+		pattern: /([\s\t]*)(?:\bglobal_defs|notification_email|notification_email_from|smtp_server|smtp_connect_timeout|router_id|virtual_server|fwmark|delay_loop|lb_algo|lb_kind|persistence_timeout|persistence_granularity|virtualhost|protocol|sorry_server|real_server|weight|TCP_CHECK|MISC_CHECK|misc_path|HTTP_GET|SSL_GET|url|path|digest|connect_port|connect_timeout|retry|delay_before_retry|vrrp_instance|state|interface|mcast_src_ip|lvs_sync_daemon_inteface|virtual_router_id|priority|advert_int|smtp_alert|authentication|auth_type|auth_pass|virtual_ipaddress|virtual_ipaddress_excluded|notify_master|notify_backup|notify_fault|vrrp_sync_group|vrrp_script|interval|script)\b/,
+		lookbehind: true
+	},
+	'constant': {
+		pattern: /(^[ \t]*)(?:rr|wrr|lc|wlc|sh|dh|lblc|NAT|DR|TUN|TCP|UDP|MASTER|BACKUP|PASS|AH)\b/,
+		lookbehind: true
+	},
+	'string': /("|')(?:\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])*\1/,
+	'email': {
+		pattern: /(?:[\w-])+@(?:[\w-])+(?:(?:\.[\w-]{2,3}){1,2})/,
+		alias: 'string',
+	},
+	'path': {
+		pattern: /(?:\/(?:[^\/\s]+\/)*[^\/\s]*)|(?:[a-zA-Z]:\\(?:\w+\\)*(?:\w+\.\w+)?)/,
```

Comment:
This is supposed to capture unix-like and windows-like file/directory names, right? Unix-like paths look kinda fine but as for windows: Why not allow any characters like in unix-like paths like this: `[a-zA-Z]:\\(?:[^\\\s]+\\)*[^\\\s]*`?

Reply:
Yes. Keppalived can't support Windows not, but:

1. It probably support in future.
2. This pattern can also be used in nginx.

The windows-like regex is copied from my previous code, I think your opinion is right. I change it now.